app/models/proposal.rb

```diff
@@ -115,6 +112,7 @@ def as_indexed_json(params = {})
       if client_data
         json[:client_data] = client_data.as_indexed_json
       end
+      json[:requester] = requester.display_name
```

Comment:
while we are working on this, would it be possible to change the "Card holder name" label to read "Card hold name or email address" ? Raphy and Amber asked me what "Card holder name" field was searching last week and I had to look at the code to figure out what it was